jitouch/Jitouch/Gesture.m

```diff
@@ -3079,7 +3078,6 @@ - (void)reload {
             MTRegisterContactFrameCallback(device, trackpadCallback);
```

Comment:
same here, add it after `familyID == 105 // macbook with touch bar`

Reply:
done